compiler/code_generator.ml

```diff
@@ -26,11 +26,17 @@ let rec comma_separate_list (expr_list : Jast.expr list) = match expr_list
     else
       (handle_expression head)
 
-and handle_expression (expr : Jast.expr) = match expr 
+and handle_expression (expr : Jast.expr) = match expr
   with Call(func_name, expr_list) -> (match func_name
     with "print" ->
       "System.out.println(" ^ (handle_expression (List.hd expr_list)) ^ ");\n"
-    | _ -> func_name ^ "(" ^ comma_separate_list expr_list ^ ")")
+    | "length" -> (match List.hd expr_list
```

Comment:
mhm! because `a.length()` carries object-oriented connotations, which we want to avoid. we decided `length(a)` follows the principle of least surprise bc it's how other functions (incl. `print`) work in our language